server/management/commands/import_uc_events.py

```diff
@@ -38,6 +38,7 @@ def handle(self, *args: Any, **options: Any) -> None:
                     "start_date": tournament["start"],
                     "end_date": tournament["end"],
                     "ultimate_central_slug": tournament["slug"],
+                    "location": tournament["Location"]["formatted_address"],
```

Comment:
Why the formatted address? Is there no better location info?

Reply:
```
"Location": {
                "model": "location",
                "id": 732849,
                "address_1": null,
                "address_2": null,
                "locality": "Bangalore",
                "region": "Karnataka",
                "postal_code": null,
                "country": "IN",
                "latitude": null,
                "longitude": null,
                "timezone": null,
                "formatted_address": "Bangalore, Karnataka, IN",
                "is_manual_entered": true,
                "created_at": "2021-01-01 07:24:17",
                "updated_at": "2021-01-01 07:24:17"
            },
```

This is the example data we get. In this the most complete one is formatted_address for most

Reply:
Do you think they'll ever start adding lat/lng? We can do funky map things if we have that info... but I guess for most things we won't?

Reply:
I guess we can add lat/lng when we need it, from UC data or doing some parsing on our end, maybe. :shrug:

Reply:
So you are suggesting to add that column now itself and use when needed right?

Reply:
But yes agree we can do some fun stuff with that. But not sure if well do it immediatly now

Reply:
Let's do it later.